compose/src/main/java/com/bottlerocketstudios/compose/devoptions/DevOptions.kt

```diff
@@ -0,0 +1,317 @@
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.bottlerocketstudios.compose.R
+
+data class DevOptionsState(
+    val environmentNames: State<List<String>>,
+    val environmentSpinnerPosition: State<Int>,
+    val baseUrl: State<String>,
+    val appVersionName: String,
+    val appVersionCode: String,
+    val appId: String,
+    val buildIdentifier: String,
+    val onEnvironmentChanged: (Int) -> Unit,
+    val onRestartCtaClick: () -> Unit,
+    val onForceCrashCtaClicked: () -> Unit
+)
+
+@Composable
+fun DevOptionsScreen(state: DevOptionsState) {
+    Scaffold(floatingActionButton = { FabLayout(state.onRestartCtaClick) }) {
+        Column(
+            modifier = Modifier.fillMaxSize()
+        ) {
+            DropdownEnvMenu(state)
+            ScrollContent(state)
+        }
+    }
+}
+
+@Composable
+private fun FabLayout(onFabClick: () -> Unit) {
+    FloatingActionButton(
+        onClick = { onFabClick() },
+        backgroundColor = MaterialTheme.colors.secondary,
+        content = {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_restart_24),
+                contentDescription = "Restart",
+                tint = Color.White
+            )
+        }
+    )
+}
+
+@Composable
+private fun ScrollContent(state: DevOptionsState) {
+    Column {
+        LazyColumn(
+            modifier = Modifier
+                .padding(start = 12.dp, end = 12.dp)
+                .fillMaxHeight()
+                .fillMaxWidth()
+                .weight(1F)
+        ) {
+            item {
+                CardDivider(Color.Transparent)
+                CardLayout { EnvironmentCard(baseUrl = state.baseUrl.value) }
+                CardDivider(MaterialTheme.colors.primary)
+            }
+            item {
+                CardLayout { AppInfoCard(state = state) }
+                CardDivider(MaterialTheme.colors.primary)
+            }
+            item {
+                CardLayout { MiscFunctionalityCard(onForceCrashCtaClick = state.onForceCrashCtaClicked) }
+                CardDivider(Color.Transparent)
+            }
+        }
+    }
+}
+
+@Composable
+private fun BottomButton(buttonText: String, onClick: () -> Unit) {
+    Button(
+        onClick = { onClick() },
+        modifier = Modifier
+            .padding(8.dp)
```

Comment:
We could possibly setup a Dimens and custom theme for next PR, this would setup app for scaling away some padding on smaller devices, or splitscreen.

Reply:
Yes! My next PR will be setting up material theme, dimens, and some standard widgets for shared and default styling.